packages/oscal-react-library/src/components/OSCALProfile.test.js

```diff
@@ -1,41 +1,38 @@
 import React from "react";
 import { render, screen, fireEvent } from "@testing-library/react";
-import { OSCALProfileLoader } from "./OSCALLoader";
 import OSCALProfile from "./OSCALProfile";
 import { parentUrlTestData } from "../test-data/Urls";
 import profileTestData from "../test-data/ProfileData";
 
-test("OSCALProfile loads", () => {
-  render(<OSCALProfileLoader />);
-});
-
-function profileRenderer() {
-  render(
-    <OSCALProfile
-      profile={profileTestData}
-      parentUrl={parentUrlTestData}
-      onResolutionComplete={() => {}}
-    />
-  );
-}
+describe("The OSCAL Profile", () => {
+  it("displays controls", async () => {
+    render(
+      <OSCALProfile
+        profile={profileTestData}
+        parentUrl={parentUrlTestData}
+        onResolutionComplete={() => {}}
+      />
+    );
 
-function testOSCALProfile(parentElementName, renderer) {
-  test(`${parentElementName} displays controls`, async () => {
-    renderer();
     const result = await screen.findByText("AC-1", {
       timeout: 30_000,
     });
     expect(result).toBeVisible();
-  });
+  }, 30_000);
```

Comment:
If we have tests failing, I think this increases the timeout for a given test.

I wonder if the test timeout needed to be >= the findByText timeout.